packs/iconics/samo-level-3.json

```diff
@@ -0,0 +1,4204 @@
t more often than they drew them together.</p><p>Samo’s ability to see what others could not made her into a healer and counselor, the person others called on when they suspected that unusual events were the doings of unquiet spirits. When fires raged, she spoke to the spirits of flame and calmed them. When people disappeared along the coastline, she would accompany the hunters who sought out the kushtaka villages hidden nearby, speaking to the apparitions that dwelt in the rivers and ocean waves to help guide them to those enclaves of lost souls.</p><p>It seemed to Samo that hardly any time at all had passed between that day on the beach where she returned her father and uncles from the kushtaka and the day when she looked into a small hand mirror and realized that many silver strands now danced amid the raven-wing black of her hair. Where had the days gone? Samo did not regret her service, but she envied the stories she heard of her old friend Nankou and his adventures slaying linnorms and rescuing would-be heroes.</p><p>One night, Samo sat amid the grove of Seer’s Home. Her grandmother had passed almost twenty years before, but the woman’s spirit still lingered in that peaceful grove, appearing to Samo’s eyes as a majestic elk woven from the leaves and grasses of the grove itself. That evening, they conversed as they had so many times before, but unlike those previous occasions, their conversation was interrupted by a spectacular and terrifying sight.</p><p>Two great figures battled in the sky, gods of the southern lands. One, covered in armored spikes, was struck down, his body ripped asunder. As red and silver droplets rained from the sky, Samo’s spiritual sight and physical sight crashed together.</p><p>As the god died above, she saw another battle: a powerful young man from a distant land fighting a fearsome demonic owl, until both were struck by the droplets of falling god-blood. The twined pair were spun a great distance through the air until Samo’s spirit sight faded and she saw that they were both truly there, in the flesh, crashing to the ground on Icemark’s shores.</p><p>A Calling thrummed within her. She knew this was a moment where all that she was and all that she had learned would be tested, when her choices and actions would determine not just the future of Icemark, but perhaps that of all the Inner Sea. Perhaps more.</p><p>She dashed to her home, and pulled on her well-used armor of bone, hide, and driftwood. She slid her clan hat down over her antlers, so that whoever she met would know that they faced a Varki woman of standing and power. Her grandmother’s apparition flowed into her, filling her with the peaceful magic of healing and quiet places.</p><p>Moving swiftly towards the shoreline and the fallen figures, her grandmother’s thoughts echoed her own. Let us go, the women thought. <em>This Calling is for us, this time is ours, and whatever may happen next, the world will know the power of the women of the north.</em></p>",
```

Comment:
```suggestion
                "backstory": "<p>\"Don't play near the water by yourself or the kushtaka will get you!\"</p><p>\"Don't wander into the trees by yourself or you'll be taken by the little-men-in-the-woods!\"</p><p>Samo had heard such warnings her entire life, repeated in fireside tales and stories told to pass the hours when her people left the summer fishing camps to travel to their inland lodges closer to Icemark's caribou herds for the winter. Samo was spirit-touched, born with the mark of the elk, an angel's golden eyes, and unusual insight. The cautionary stories always seemed to be directed at her, specifically, when it was her father's turn to tell the tales, as though her gifts would be more likely to lead her astray than the other children. Perhaps there was some merit to that worry.</p><p>It wasn't that she didn't believe the stories on that summer day when she crept down to the shoreline by herself. Her father and uncles had been gone for several weeks now, taking her grandmother to Seer's Home, where the elderly woman would live out the rest of her days among the Varki elders, now that her aged hips and back could no longer handle the nomadic migrations between fishing camps and hunting lodges. Samo wanted to be the first to greet her father on his return, and she was twelve years old now, nearly an adult by her reckoning, and she was accustomed to the little spirits warning her of dangers before others ever became aware of them.</p><p>Thus, it was particularly surprising when a cold, slimy paw wrapped in dank fur clutched her from behind, covering her mouth. Her screams were completely muffled, and all she could do was struggle in terror as she realized what had grabbed her: a kushtaka! Part man, part otter, the soulless creatures would drag her back to their village where they would cast her soul from her body and transform her into one of them! Worse yet, there were nearly twenty of the fearsome creatures quietly slipping from the waters around her. An entire kushtaka raiding party had come, doubtless here to take the encampment and disappear with them before anyone could so much as raise an alarm.</p><p>Despite all the terror she already felt, Samo discovered that things could still get worse. For every kushtaka that slipped from the waves, the insubstantial, sad-eyed form of a human walked beside them. Among the lost souls, she saw her father and uncles. Thrashing desperately, Samo thrust her small hand out towards her father's spirit, and the ghostly figure reached out to her in response. As their fingertips brushed, it felt like lightning struck.</p><p>The kushtaka holding Samo screamed in agony, the sound finally awakening the people in the nearby fishing camp. Her father's ghostly figure was nowhere to be seen, but the monster's body changed and contorted, its otter's fangs and fur falling away as its bones stretched to reveal her father's features.</p><p>Samo used the opportunity to run towards the camp, where she snatched up one of the nets laced with dog bone that had been left there to repel just such an incursion. Dashing towards the kushtaka, she saw the spirits of the lost Varki moving to aid her, and she swung the net about, pulling the spirits together with the soulless bodies that had once belonged to them, triggering transformations like the one that had happened with her father. The sounds and cries of battle finally drew the rest of the village to her side, and Varki men and women with spears and daggers did what they could to fend the remaining kushtaka away while herding as many as they could towards Samo and her net.</p><p>Though over half of the kushtaka escaped back into the waves, Samo managed to save her father, her uncles, and a handful of other Varki from neighboring encampments. As Samo's people gathered around, her father knelt down before her. \"You do not belong here anymore, beloved. Your gifts must be nurtured by those who understand them.\"</p><p>Samo joined her grandmother at Seer's Home and helped care for the Varki elders there while they taught her everything they knew of her people's legends, traditions, and magic. In her early twenties she had a brief, passionate love with a handsome young man named Nankou who also resided at Seer's Home for a time, but their duties called them apart more often than they drew them together.</p><p>Samo's ability to see what others could not made her into a healer and counselor, the person others called on when they suspected that unusual events were the doings of unquiet spirits. When fires raged, she spoke to the spirits of flame and calmed them. When people disappeared along the coastline, she would accompany the hunters who sought out the kushtaka villages hidden nearby, speaking to the apparitions that dwelt in the rivers and ocean waves to help guide them to those enclaves of lost souls.</p><p>It seemed to Samo that hardly any time at all had passed between that day on the beach where she returned her father and uncles from the kushtaka and the day when she looked into a small hand mirror and realized that many silver strands now danced amid the raven-wing black of her hair. Where had the days gone? Samo did not regret her service, but she envied the stories she heard of her old friend Nankou and his adventures slaying linnorms and rescuing would-be heroes.</p><p>One night, Samo sat amid the grove of Seer's Home. Her grandmother had passed almost twenty years before, but the woman's spirit still lingered in that peaceful grove, appearing to Samo's eyes as a majestic elk woven from the leaves and grasses of the grove itself. That evening, they conversed as they had so many times before, but unlike those previous occasions, their conversation was interrupted by a spectacular and terrifying sight.</p><p>Two great figures battled in the sky, gods of the southern lands. One, covered in armored spikes, was struck down, his body ripped asunder. As red and silver droplets rained from the sky, Samo's spiritual sight and physical sight crashed together.</p><p>As the god died above, she saw another battle: a powerful young man from a distant land fighting a fearsome demonic owl, until both were struck by the droplets of falling god-blood. The twined pair were spun a great distance through the air until Samo's spirit sight faded and she saw that they were both truly there, in the flesh, crashing to the ground on Icemark's shores.</p><p>A Calling thrummed within her. She knew this was a moment where all that she was and all that she had learned would be tested, when her choices and actions would determine not just the future of Icemark, but perhaps that of all the Inner Sea. Perhaps more.</p><p>She dashed to her home, and pulled on her well-used armor of bone, hide, and driftwood. She slid her clan hat down over her antlers, so that whoever she met would know that they faced a Varki woman of standing and power. Her grandmother's apparition flowed into her, filling her with the peaceful magic of healing and quiet places.</p><p>Moving swiftly towards the shoreline and the fallen figures, her grandmother's thoughts echoed her own. Let us go, the women thought. <em>This Calling is for us, this time is ours, and whatever may happen next, the world will know the power of the women of the north.</em></p>",
```